### ISSUE TITLE
Regression sending mail cuts off anything after a blank line

### ISSUE THREAD

**Issue by BLamy** [nylas/nylas-mail#3269]
It seems whenever I send mail (gmail) if that email has a blank line. It will insert the 'Sent from Nylas Mail, the best free email app for work' in that blank line. All text after that is truncated.

This is extremely frustrating because you type out the email everything looks fine and then you hit send and the email it sends is different from what you typed. Any text that was after a blank line is permanently lost. 

Unsure if this is specific to me but this happened on all 3 emails I sent immediately after updating today.

**Comment by BLamy** [comment]
For instance if I paste that previous comment into my nylas and then click send it sends the following

```
It seems whenever I send mail (gmail) if that email has a blank line. It will insert the 'Sent from Nylas Mail, the best free email app for work' in that blank line. All text after that is truncated.
Sent from Nylas Mail, the best free email app for work
```

**Comment by braverun** [comment]
You're not alone. This same thing happened to me twice.

**Comment by martylouis** [comment]
This just happened to me as well. Typed out a multi paragraph email, only sent the barely first two lines.

> Hi Client,
> By now you should have received a link from

All that was sent. 

- Nylas Version v1.0.21
- macOS v10.12.4 Beta (16E144f) 
- 3rd Party plugins: n1-avatars

**Comment by jstejada** [comment]
thanks for the report all, we are looking into this issue

**Comment by daviestar** [comment]
"Me too"

I've just sent an email, then upon checking it thought "WTF?", and thinking it was my error, rewrote it and sent with an apology, and that was also cut off!!

Having to use mail.google.com....

**Comment by jstejada** [comment]
this should be fixed on 1.0.26 which was released today! Thanks for your patience!
